docs/FAQ.md

```diff
@@ -27,28 +27,28 @@ these changes and release updates as quickly as possible.
 ### Which distributions are supported?
 
 The RAPIDS Accelerator for Apache Spark officially supports
-[Apache Spark](get-started/getting-started.md),
-[Databricks Runtime 7.0](get-started/getting-started-with-rapids-accelerator-on-databricks.md)
-and [Google Cloud Dataproc](get-started/getting-started-gcp.md).
+[Apache Spark](get-started/getting-started-on-prem),
+[Databricks Runtime 7.0](get-started/getting-started-databricks)
+and [Google Cloud Dataproc](get-started/getting-started-gcp).
```

Comment:
If you don't put in the `.md` github cannot find them.

https://github.com/NVIDIA/spark-rapids/blob/990f7ecdb466293990d8cd57aa886d84587b8f89/docs/get-started/getting-started-on-prem

returns a 404

I thought github pages is smart enough to strip them out for you.

This applies everywhere

Reply:
Added the .md links back.  Testing locally with Jekyll they do not work.  I will have to test again with gh-pages.

Reply:
Yes we have run into this every time someone wants to update the docs.  It does not work with Jekyll so they rip them out, but it brakes regular github, and it works just fine on github pages.

Note that if you include the .md then the link on github pages is back to the markdown file on the master branch instead of the html file.